docs: fix breaking URL for RxJS marble testing

OmarioHassan opened the pull request:
When checking this URL for the `RxJS marble testing` [here](https://angular.io/guide/testing-components-scenarios#component-testing-scenarios)  I've found it pointing to `Page not found` so I'm tried to update the docs

## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A


## What is the new behavior?


## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

ajitsinghkaler reviewed (changes requested):
@OmarioHasan  Everything seems fine.
Except for your commit message body contains lines greater than 120 characters.
Can you reduce the line length to less than 120 characters?

OmarioHasan commented:
Sure
thanks for reply

ajitsinghkaler reviewed (approved):
LGTM

gkalpak commented on aio/content/guide/testing-components-scenarios.md:
```diff
@@ -779,7 +779,7 @@ which reports router activity, is a _hot_ observable.
 RxJS marble testing is a rich subject, beyond the scope of this guide.
 Learn about it on the web, starting with the
-[official documentation](https://github.com/ReactiveX/rxjs/blob/master/doc/writing-marble-tests.md).
+[official documentation](https://github.com/ReactiveX/rxjs/blob/master/docs_app/content/guide/testing/marble-testing.md).
```
I think https://rxjs.dev/guide/testing/marble-testing would be a better URL to use here.

OmarioHasan replied:
done
thanks for adding this link

ajitsinghkaler commented:
You need to squash your commits

OmarioHassan committed:
docs: fix breaking URL for RxJS marble testing

When checking this URL for the `RxJS marble testing` Ive found it pointing to `Page not found`

gkalpak reviewed (approved):
LGTM :+1:
Thx, @OmarioHasan!

Reviewed-for: global-docs-approvers
